marlowe-website/src/index.js

```diff
@@ -153,7 +154,51 @@ function fixCrossLinks() {
   }
 }
 
+function activateYoutube() {
+  $(".video-container iframe").each((_, elm) => {
+    elm.setAttribute("src", elm.getAttribute("data-src"));
+    elm.classList.remove("hidden");
+  });
+  $(".video-container p").each((_, elm) => {
+    elm.classList.add("hidden");
+    elm.classList.remove("flex");
+  });
+}
+function activateTypeform() {
+  var script = document.createElement("script");
+  script.type = "text/javascript";
+  script.async = !0;
```

Comment:
Isn't `!0` the same as `true`?

Reply:
I'd assume so... I just copy and pasted from the google analytics code. Maybe they used it to save some bytes as `!0` is smaller than `true`